Project-Uni/RootShare
# Admin is set as member of community

**Updates**
1. Admin gets set as community member by default for accurate counts
2. When editing a community, the new admin gets added as a member to the community if they are not already there

**Testing**
- Create a new community, and set the admin. Go into the database, the id of the admin should be in the list of members
- Change the admin of a community. The new admin should be added to the list of members
